packages/test/jest.test.js

```diff
@@ -0,0 +1,93 @@
+/* eslint-env jest */
+
+const hrp = require("http-request-plus").default;
+const jrp = require("json-rpc-protocol");
+
+async function call(method, params) {
+  try {
+    return jrp.parse.result(
+      await hrp
+        .post("http://localhost:4000/api/", {
+          body: jrp.format.request(0, method, params),
+        })
+        .readAll("utf-8")
+    );
+  } catch (error) {
+    return error;
+  }
+}
+
+test("create entry", async () => {
+  expect(
+    await call("createEntry", { name: "name1", content: "content1" })
+  ).toEqual(1);
+  expect(await call("createEntry", { content: "content2" })).toEqual(2);
+  expect(await call("createEntry", { name: "name3" })).toEqual(3);
+
+  const response = await call("listEntries");
+
+  expect(response[0].name).toEqual("name1");
+  expect(response[0].content).toEqual("content1");
+  expect(response[1].content).toEqual("content2");
+  expect(response[2].name).toEqual("name3");
```

Comment:
```js
expect(response).toEqual([
  {
    id: 1,
    name: "name1",
    content: "content1",
  },
  {
    id: 2,
    content: "content2",
  },
  {
    id: 3,
    name: "name3",
  },
])
```

Reply:
We can't do like that, there are two others parameters to precise in this case: "created" and "updated". But we don't know what is the value of its each time we run the tests.

Reply:
Almost straight from [the documentation](https://jestjs.io/docs/en/expect#expectanyconstructor):

```js
expect(response).toEqual([
  {
    id: 1,
    name: "name1",
    content: "content1",
    created: expect.any(Number),
    updated: expect.any(Number),
  },
  {
    id: 2,
    content: "content2",
    created: expect.any(Number),
    updated: expect.any(Number),
  },
  {
    id: 3,
    name: "name3",
    created: expect.any(Number),
    updated: expect.any(Number),
  },
]);
```